### ISSUE TITLE
Adding missing prefix via quickfix inserts garbage

### ISSUE THREAD

**Issue by atextor** [AKSW/Xturtle#69]
To reproduce the error,
- start with a blank .ttl file,
- enter something like `[] a rdfs:Class .`
- click on the quickfix "add prefix rdfs" for the missing prefix.

The expected output is:

```
@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> .
[] a rdfs:Class.
```

The actual output is:
`@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> DEFINEDINCUSTOMLEXER[] a rdfs:Class.`

This is with the latest snapshot version.

**Comment by nittka** [comment]
Thanks for the report. This looks like a side effect of grammar changes made in order to allow '.' within local names. I cannot predict when this bug will be fixed but it is top of the list.

**Comment by nittka** [comment]
Fixed in 1.1.1-SNAPSHOT

**Comment by nittka** [comment]
I seem not to be able to assign a milestone to the issue!?
